.github/workflows/ci.yml

```diff
@@ -224,9 +224,8 @@ jobs:
                     git clone https://github.com/spryker/docker-sdk.git ./docker
                     docker/sdk boot deploy.ci.api.mariadb.yml
                     docker/sdk up -t
-                    docker/sdk cli composer dump-autoload -o -a --apcu
                     docker/sdk testing codecept fixtures
-                    docker/sdk testing console queue:worker:start --stop-when-empty
+                    docker/sdk console queue:worker:start --stop-when-empty
```

Comment:
What is the reason this is changed from testing to console? Please add a comment to this line so everyone knows why this line is different from the others.

I would also move it below the other testing ones to have them clearly separated and not mixed together.

Reply:
requirements

Reply:
execly testing uses more services. We don't need them for this command